microshift_updating/microshift-about-updates.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: ASSEMBLY
+[id="microshift-about-updates"]
+=  About {product-title} updates
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-about-updates
+
+toc::[]
+
+You can update a {product-title} cluster by using the OpenShift CLI (`oc`).
+// This PR is for the book build. Note that the OCP structure consists of a landing page of xrefs to other major sections within the book. MicroShift likely does not require that depth of structure, so starting simply with one page.
+
+[id="microshift-about-updates-understanding-microshift-updates"]
+== Understanding {product-title} updates
+{product-title} updates are available as either RPMs or by embedding the {product-title} image in an RPM OSTree blueprint.
+You can update an {product-title} cluster by using the OpenShift CLI (`oc`).
+//Platform administrators can view new update options by looking at the output of the `oc adm upgrade` command.
+//An update begins when...
+
+[NOTE]
+====
+Operators previously installed must be reinstalled using manifests.
+====
+
+=== Using RPM updates
+Using the RPM update method replaces your existing version. No rollback is possible with this update type.
+//we can call a module here or xref out; not sure the best method for our use case until we have the content
+
+=== Using RPM OSTree updates
+Using the RPM OSTree update path allows for system rollback.
+//we can call a module here or xref out; not sure the best method for our use case until we have the content
```

Comment:
These will be their own module since they'll likely (?) be a procedure for each one (I know you already know that, just thinking through this). That said, how I'm currently envisioning this is that this assembly is information about updating, and that you'll have a second assembly nested under "Updating clusters" for the actual procedures? In which case, it would be an xref.